server/helpers/serverResponse.js

```diff
@@ -0,0 +1,10 @@
+/**
```

Comment:
please remove this file

Reply:
I have updated my PR, and I noticed that the file has been updated on my branch as well. Basically, my local branch is in sync with the remote develop, so this file is no longer a repeated file.

You would notice that this file is no longer in the `Files changed` section of this PR.